andela/ah-the-answer-backend
# #165243774 Fix  email verification url

__What does this PR Do?__
Fix a bug that prevents users from verifying their accounts when they click the email verification link
__How should this be manually tested?__
- Clone the repo and navigate to the root directory
- Create a virtual environment by typing` virtualenv venv` and activate it
- Install requirements by typing `pip install -r requirements.txt`

Setup a database and define the DATABASE_URL in a .env file. Export the configuration variable
Run the tests by typing python `manage.py test`

__To test on postman do the following__
- Run the Django server to serve up the app by typing 
```python
python manage.py runserver
```
- Using postman, create a user by sending a POST  request as shown below.

<img width="1105" alt="Screenshot 2019-04-10 at 15 45 10" src="https://user-images.githubusercontent.com/22986772/55879614-ba03fb80-5ba7-11e9-9562-58070a9178a7.png">

- Check the inbox of the email provided above and click the confirm email button as shown below

<img width="909" alt="Screenshot 2019-04-10 at 15 54 30" src="https://user-images.githubusercontent.com/22986772/55880251-fbe17180-5ba8-11e9-8495-3e3947eb1ab2.png">

- You should be redirected to a page with the message bellow
```
{"user": {"Success": "Your email has been verified"}}
```





__Relevant PT stories__
https://www.pivotaltracker.com/story/show/165243774
